OptionSelector/Countries.h

```diff
@@ -0,0 +1,14 @@
+//
+//  Countries.h
+//  OptionSelector
+//
+//  Created by Charles Kang on 8/11/15.
+//  Copyright (c) 2015 Mike Kavouras. All rights reserved.
+//
+
+#import "Categories.h"
+#import <Foundation/Foundation.h>
+
+@interface Countries : Categories
```

Comment:
You should only use a subclass if the behavior of the subclass is different. In this case, only the data is different. This could be accomplished more simply by initializing all your categories in one place instead of splitting them into subclasses like this.